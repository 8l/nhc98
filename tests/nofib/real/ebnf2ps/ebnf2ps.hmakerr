Can't find module PrelBase in user directories
	.
  Or in installed libraries/packages at
	/usr/malcolm/local/include/nhc98
	/usr/malcolm/local/include/nhc98/packages/base
  Asked for by: HappyParser.ly
  Fix using the -I, -P, or -package flags.

Stop - hmake dependency error.
