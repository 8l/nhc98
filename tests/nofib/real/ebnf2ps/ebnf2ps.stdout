%!PS-Adobe-1.0
%%DocumentFonts: Times-Roman
%%Title: apat
%%Creator: ebnf2ps (Copyright 1994 by Peter Thiemann)
%%Pages: 0
%%BoundingBox: 0 0 169 382
%%EndComments
/$Ebnf2psDict 100 dict def
$Ebnf2psDict begin
/l {lineto} bind def
/m {moveto} bind def
/rl {rlineto} bind def
/rm {rmoveto} bind def
/s {stroke} bind def
/n {newpath} bind def
/gs {gsave} bind def
/gr {grestore} bind def
/clp {closepath} bind def
/slw {setlinewidth} bind def
/graycol {dup dup currentrgbcolor 4 -2 roll mul 4 -2 roll mul
4 -2 roll mul setrgbcolor} bind def
/scol {3 {255 div 3 1 roll} repeat setrgbcolor} bind def
 /apr {arcto 4 {pop} repeat} def
/Box {
  /linewidth exch def
  linewidth sub /height exch def
  linewidth sub /width exch def
   n m
  width 0 rl
  0 height rl
  width neg 0 rl
  0 height neg rl
  clp linewidth slw s
} def
 /RBox {
  /linewidth exch def
  /height exch def
  /width exch def
  /lly exch def
  /llx exch def
  linewidth 2 div dup llx add /llx exch def lly add /lly exch def
  /height height linewidth sub def
  /width  width  linewidth sub def
  /height2 height 2 div def
  /width2  width  2 div def
  /urx llx width add def
  /ury lly height add def
  /mmx llx width2 add def
  /mmy lly height2 add def
  /radius width2 height2 ge {height2} {width2} ifelse def
   n mmx lly m
  urx lly urx mmy radius apr
  urx ury mmx ury radius apr
  llx ury llx mmy radius apr
  llx lly mmx lly radius apr
  mmx lly l
  clp linewidth slw s
} def
end
/$Ebnf2psBegin {$Ebnf2psDict begin /$Ebnf2psEnteredState save def} def
/$Ebnf2psEnd {$Ebnf2psEnteredState restore end} def

%%EndProlog

$Ebnf2psBegin
 0 0 0 scol 0.30 slw
n 8 182.75 m 2 -2 rl -2 -2 rl s
n 0 180.75 m 10 0 rl s
/Times-Roman findfont 10 scalefont setfont
 0 373.17 m(apat) show
n 166.60 182.75 m 2 -2 rl -2 -2 rl s
n 163.60 180.75 m 5 0 rl s
n 153.60 9 m 0 0 rl s
n 134.88 53 m 18.72 0 rl s
n 126.32 110.50 m 27.28 0 rl s
n 126.32 132.50 m 27.28 0 rl s
n 84.88 154.50 m 68.72 0 rl s
n 55.66 176.50 m 97.94 0 rl s
n 54.78 198.50 m 98.82 0 rl s
n 49.21 220.50 m 104.39 0 rl s
n 59.77 242.50 m 93.83 0 rl s
n 46.44 264.50 m 107.16 0 rl s
n 37 286.50 m 116.60 0 rl s
n 66.63 308.50 m 86.97 0 rl s
n 100.20 330.50 m 53.40 0 rl s
n 44.77 352.50 m 108.83 0 rl s
n 153.60 53 m 5 0 rl s
n 153.60 110.50 m 5 0 rl s
n 153.60 132.50 m 5 0 rl s
n 153.60 154.50 m 5 0 rl s
n 153.60 176.50 m 5 0 rl s
n 153.60 198.50 m 5 0 rl s
n 153.60 220.50 m 5 0 rl s
n 153.60 242.50 m 5 0 rl s
n 153.60 264.50 m 5 0 rl s
n 153.60 286.50 m 5 0 rl s
n 153.60 308.50 m 5 0 rl s
n 153.60 330.50 m 5 0 rl s
n 15 53 m 5 0 rl s
n 15 110.50 m 5 0 rl s
n 15 132.50 m 5 0 rl s
n 15 154.50 m 5 0 rl s
n 15 176.50 m 5 0 rl s
n 15 198.50 m 5 0 rl s
n 15 220.50 m 5 0 rl s
n 15 242.50 m 5 0 rl s
n 15 264.50 m 5 0 rl s
n 15 286.50 m 5 0 rl s
n 15 308.50 m 5 0 rl s
n 15 330.50 m 5 0 rl s
n 158.60 14 m 0 333.50 rl s
n 153.60 9 m 158.60 9 158.60 14 5 apr
 158.60 14 l s
n 158.60 347.50 m 158.60 352.50 153.60 352.50 5 apr
 153.60 352.50 l s
n 15 14 m 0 333.50 rl s
n 20 9 m 15 9 15 14 5 apr
 15 14 l s
n 15 347.50 m 15 352.50 20 352.50 5 apr
 20 352.50 l s
n 158.60 180.75 m 5 0 rl s
n 10 180.75 m 5 0 rl s
 138.27 0.50 15.33 17 1 RBox
 144.27 6.17 m(\)) show
 0.30 slw
n 136.27 11 m 2 -2 rl -2 -2 rl s
n 133.27 9 m 5 0 rl s
n 128.27 14 m 0 12 rl s
n 94.05 14 m 0 12 rl s
n 128.27 26 m 128.27 31 123.27 31 5 apr
 123.27 31 l s
n 123.27 9 m 128.27 9 128.27 14 5 apr
 128.27 14 l s
n 94.05 26 m 94.05 31 99.05 31 5 apr
 99.05 31 l s
n 99.05 9 m 94.05 9 94.05 14 5 apr
 94.05 14 l s
n 118.41 31 m 4.86 0 rl s
n 99.05 31 m 4.86 0 rl s
n 123.27 9 m 10 0 rl s
n 89.05 9 m 10 0 rl s
n 120.41 33 m -2 -2 rl 2 -2 rl s
 103.91 22.50 14.50 17 1 RBox
 109.91 28.17 m(,) show
 99.05 0.50 24.22 17 1 Box
 105.05 6.17 m(pat) show
 0.30 slw
n 87.05 11 m 2 -2 rl -2 -2 rl s
n 84.05 9 m 5 0 rl s
 69.55 0.50 14.50 17 1 RBox
 75.55 6.17 m(,) show
 0.30 slw
n 67.55 11 m 2 -2 rl -2 -2 rl s
n 64.55 9 m 5 0 rl s
 40.33 0.50 24.22 17 1 Box
 46.33 6.17 m(pat) show
 0.30 slw
n 38.33 11 m 2 -2 rl -2 -2 rl s
n 35.33 9 m 5 0 rl s
 20 0.50 15.33 17 1 RBox
 26 6.17 m(\() show
 119.55 44.50 15.33 17 1 RBox
 125.55 50.17 m(]) show
 0.30 slw
n 117.55 55 m 2 -2 rl -2 -2 rl s
n 114.55 53 m 5 0 rl s
n 109.55 58 m 0 3.50 rl s
n 45.33 58 m 0 3.50 rl s
n 99.55 66.50 m 5 0 rl s
n 109.55 61.50 m 109.55 66.50 104.55 66.50 5 apr
 104.55 66.50 l s
n 114.55 53 m 109.55 53 109.55 58 5 apr
 109.55 58 l s
n 50.33 66.50 m 5 0 rl s
n 45.33 61.50 m 45.33 66.50 50.33 66.50 5 apr
 50.33 66.50 l s
n 40.33 53 m 45.33 53 45.33 58 5 apr
 45.33 58 l s
n 40.33 53 m 74.22 0 rl s
n 53.33 68.50 m 2 -2 rl -2 -2 rl s
n 94.55 71.50 m 0 12 rl s
n 60.33 71.50 m 0 12 rl s
n 94.55 83.50 m 94.55 88.50 89.55 88.50 5 apr
 89.55 88.50 l s
n 89.55 66.50 m 94.55 66.50 94.55 71.50 5 apr
 94.55 71.50 l s
n 60.33 83.50 m 60.33 88.50 65.33 88.50 5 apr
 65.33 88.50 l s
n 65.33 66.50 m 60.33 66.50 60.33 71.50 5 apr
 60.33 71.50 l s
n 84.69 88.50 m 4.86 0 rl s
n 65.33 88.50 m 4.86 0 rl s
n 89.55 66.50 m 10 0 rl s
n 55.33 66.50 m 10 0 rl s
n 86.69 90.50 m -2 -2 rl 2 -2 rl s
 70.19 80 14.50 17 1 RBox
 76.19 85.67 m(,) show
 65.33 58 24.22 17 1 Box
 71.33 63.67 m(pat) show
 0.30 slw
n 38.33 55 m 2 -2 rl -2 -2 rl s
n 35.33 53 m 5 0 rl s
 20 44.50 15.33 17 1 RBox
 26 50.17 m([) show
 110.99 102 15.33 17 1 RBox
 116.99 107.67 m(\)) show
 0.30 slw
n 108.99 112.50 m 2 -2 rl -2 -2 rl s
n 105.99 110.50 m 5 0 rl s
 81.77 102 24.22 17 1 Box
 87.77 107.67 m(pat) show
 0.30 slw
n 79.77 112.50 m 2 -2 rl -2 -2 rl s
n 76.77 110.50 m 5 0 rl s
 40.33 102 36.44 17 1 Box
 46.33 107.67 m(conop) show
 0.30 slw
n 38.33 112.50 m 2 -2 rl -2 -2 rl s
n 35.33 110.50 m 5 0 rl s
 20 102 15.33 17 1 RBox
 26 107.67 m(\() show
 110.99 124 15.33 17 1 RBox
 116.99 129.67 m(\)) show
 0.30 slw
n 108.99 134.50 m 2 -2 rl -2 -2 rl s
n 105.99 132.50 m 5 0 rl s
 69.55 124 36.44 17 1 Box
 75.55 129.67 m(conop) show
 0.30 slw
n 67.55 134.50 m 2 -2 rl -2 -2 rl s
n 64.55 132.50 m 5 0 rl s
 40.33 124 24.22 17 1 Box
 46.33 129.67 m(pat) show
 0.30 slw
n 38.33 134.50 m 2 -2 rl -2 -2 rl s
n 35.33 132.50 m 5 0 rl s
 20 124 15.33 17 1 RBox
 26 129.67 m(\() show
 69.55 146 15.33 17 1 RBox
 75.55 151.67 m(\)) show
 0.30 slw
n 67.55 156.50 m 2 -2 rl -2 -2 rl s
n 64.55 154.50 m 5 0 rl s
 40.33 146 24.22 17 1 Box
 46.33 151.67 m(pat) show
 0.30 slw
n 38.33 156.50 m 2 -2 rl -2 -2 rl s
n 35.33 154.50 m 5 0 rl s
 20 146 15.33 17 1 RBox
 26 151.67 m(\() show
 40.33 168 15.33 17 1 RBox
 46.33 173.67 m(\)) show
 0.30 slw
n 38.33 178.50 m 2 -2 rl -2 -2 rl s
n 35.33 176.50 m 5 0 rl s
 20 168 15.33 17 1 RBox
 26 173.67 m(\() show
 20 190 34.78 17 1 Box
 26 195.67 m(string) show
 20 212 29.21 17 1 Box
 26 217.67 m(char) show
 20 234 39.77 17 1 Box
 26 239.67 m(integer) show
 20 256 26.44 17 1 Box
 26 261.67 m(con) show
 20 278 17 17 1 RBox
 26 283.67 m(_) show
 42.41 300 24.22 17 1 Box
 48.41 305.67 m(pat) show
 0.30 slw
n 40.41 310.50 m 2 -2 rl -2 -2 rl s
n 37.41 308.50 m 5 0 rl s
 20 300 17.41 17 1 RBox
 26 305.67 m(~) show
 75.98 322 24.22 17 1 Box
 81.98 327.67 m(pat) show
 0.30 slw
n 73.98 332.50 m 2 -2 rl -2 -2 rl s
n 70.98 330.50 m 5 0 rl s
 49.77 322 21.21 17 1 RBox
 55.77 327.67 m(@) show
 0.30 slw
n 47.77 332.50 m 2 -2 rl -2 -2 rl s
n 44.77 330.50 m 5 0 rl s
 20 322 24.77 17 1 Box
 26 327.67 m(var) show
 20 344 24.77 17 1 Box
 26 349.67 m(var) show

$Ebnf2psEnd
